userguide/contribute.rst

```diff
@@ -0,0 +1,179 @@
+.. _Contributing to %brand%:
+
+Contributing to %brand%
+=================================
+
+%brand% is an open source community, relying on the input and
+expertise of its users to help grow and improve %brand%. When users
+take time to assist the community, their contributions benefit everyone
+who uses %brand%.
+
+This section describes some areas of participation to contribute to
```

Comment:
"This section describes how to participate and contribute to FreeNAS."

Reply:
Replace all %brand% entries in this section with FreeNAS and the associate Sphinx markup for the (R) mark.
I'll review again after that.  Thanks!